### ISSUE TITLE
feature request: open the selected folder in terminal (and add a hotkey for the feature)

### ISSUE THREAD

**Issue by 07416** [cboxdoerfer/fsearch#205]
Preferably available from both the context menu and with a hotkey.

**Comment by cboxdoerfer** [comment]
Yes, that sounds useful. But I'm not sure if there's reliable way to determine the default terminal on all platforms, so users would probably need to specify their preferred terminal emulator in the preferences.
